src/main/java/racingcar/Application.java

```diff
@@ -0,0 +1,12 @@
+package racingcar;
+
+public class Application {
+    public static void main(String[] args) {
+        Console console = new Console();
+
+        RacingCarGame racingCarGame = new RacingCarGame(console.getTryCount(),
+            console.getCarNames());
```

Comment:
```suggestion
       RacingCarGame racingCarGame = new RacingCarGame(console.getTryCount(),console.getCarNames());
```

이정도 길이는 개행이 없어도 괜찮을 것 같아요 ″̮